src/plugins/intel_npu/src/plugin/npuw/llm_infer_request.cpp

```diff
@@ -100,11 +105,40 @@ void copy_columns_by_row_chunks(ov::SoPtr<ov::ITensor> src, ov::SoPtr<ov::ITenso
         std::copy_n(src_p + src_offset, chunk_byte_size, dst_p + dst_offset);
     }
 }
+
+std::optional<ov::Output<const ov::Node>> find_port_by_name(const std::vector<ov::Output<const ov::Node>>& ports,
+                                                            const std::string& name) {
+    auto it = std::find_if(ports.begin(), ports.end(), [&](const auto& port) {
+        return port.get_any_name() == name;
```

Comment:
what if a port have multiple names? In this case it may be a hit or miss, depending on that `any` returns.

Reply:
good point

Reply:
Checked all names, thx!